treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -6,25 +6,13 @@
     default_component: Performance
     default_product: Testing
     text: |
-      Talos has detected a Firefox performance regression from push:
-
-      {{ revisionHref }}
-
-      As author of one of the patches included in that push, we need your help to address this regression.
+      Perfherder has detected a performance regression from push [{{ revisionId }}]({{ revisionHref }}). As author of one of the patches included in that push, we need your help to address this regression.
```

Comment:
We're not mentioning which perf framework this originated on. How will engineers know where to navigate from the https://wiki.mozilla.org/TestEngineering/Performance/Handling_regression_bugs link?